docs/source/tutorials/stable_diffusion.mdx

```diff
@@ -124,3 +124,52 @@ pipeline = GaudiStableDiffusionPipeline.from_pretrained(
     torch_dtype=torch.bfloat16
 )
 ```
+
+# Super-resolution
+
+The Stable Diffusion upscaler diffusion model was created by the researchers and engineers from CompVis, Stability AI, and LAION. It is used to enhance the resolution of input images by a factor of 4.
+
+See [here](https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/upscale) for more information
+
+## How to upscale low resolution images?
+
+To generate RGB and depth images with Stable Diffusion Upscale on Gaudi, you need to instantiate two instances:
+- A pipeline with [`GaudiStableDiffusionUpscalePipeline`]. 
```

Comment:
Our doc builder won't be able to find a link to it automatically (this is specific to Optimum as we have subpackages). So I suggest adding a reference to it [here](https://github.com/huggingface/optimum-habana/blob/main/docs/source/package_reference/stable_diffusion_pipeline.mdx), same as for `GaudiStableDiffusionPipeline`.

Reply:
Done!